std/http/server_test.ts

```diff
@@ -529,4 +529,57 @@ test({
   }
 });
 
+test({
+  name: "[http] serveTLS",
+  async fn(): Promise<void> {
+    // Runs a simple server as another process
+    const p = Deno.run({
+      args: [
+        Deno.execPath(),
+        "http/testdata/simple_https_server.ts",
+        "--allow-net",
+        "--allow-read"
+      ],
+      stdout: "piped"
+    });
+
+    try {
+      const r = new TextProtoReader(new BufReader(p.stdout!));
+      const s = await r.readLine();
+      assert(s !== Deno.EOF && s.includes("server listening"));
+
+      let serverIsRunning = true;
+      p.status()
+        .then(
+          (): void => {
+            serverIsRunning = false;
+          }
+        )
+        .catch((_): void => {}); // Ignores the error when closing the process.
+
+      await delay(100);
```

Comment:
Is this necessary given you wait for "server listening" on stdout above?

Reply:
Without this delay we get `Connection refused` error on client side.